libs/platform/src/lib/list/base-list-item.ts

```diff
@@ -534,7 +534,9 @@ export class BaseListItem extends BaseComponent implements OnInit, AfterViewInit
      * Show navigation for single list
      */
     ngOnInit(): void {
-        this.id = `fdp-list-item-${nextListItemId++}`;
+        if (!this.id || this.id === this.defaultId) {
+            this.id = `fdp-list-item-${nextListItemId++}`;
```

Comment:
By default, id is `defaultId` which is overridden by this implementation. We provide custom input parameter for id but it always gets overridden.